.github/workflows/build-wsl.yaml

```diff
@@ -213,35 +216,41 @@ jobs:
 
           cd "${{ env.buildInfoPath }}"
           git add "${fingerprint_file}"
-          # If build-id is 0, we don’t want to submit it to the store as first submission is manual.
-          # We let the other steps and jobs happening to save artifacts (build-id, fingerprints) of the first build
+
           hasChanges="$(git diff --staged ${fingerprint_file})"
           cd -
-
           if [ -z "${hasChanges}" ]; then
             exit 0
           fi
 
-          echo "::set-output name=save-artifacts::true"
+          echo "::set-output name=has-changes::true"
```

Comment:
We should set some point to stop using `set-output` and prefer `$GITHUB_OUTPUT`, but that can be set for later.

Reply:
Agreed but for later :)